frame/timestamp/src/lib.rs

```diff
@@ -241,6 +241,10 @@ pub mod pallet {
 				Ok(())
 			}
 		}
+
+		fn is_inherent(call: &Self::Call) -> bool {
```

Comment:
I would expect this to be something like `#[inherent]` as a tag over the function

Reply:
yes, we could also move all is_inherent_required, create_inherent and check_inherent implementation over the function.

I think the general story of improving declaration of inherent and validateunsigned is doable. (though in another PR)